### ISSUE TITLE
DiffAugment Forward Pass

### ISSUE THREAD

**Issue by shreejalt** [lucidrains/lightweight-gan#95]
Hi @lucidrains, 
Thanks a lot for your work. I had one doubt regarding your implementation of DiffAugment.

When you train the discriminator, you pass in D_aug the fake image using detach=True. Can you explain why you do that?

Also, when you pass the real image in D_aug, you don't do the same?

Is it because you pass the requires_grad=True variable i.e generated image and you don't want to pass the gradients through the generator?

Thanking you in advance
